### PR TITLE
bandaid fixture couch issue

### PR DESCRIPTION
**Pull request by sravfeyn**
https://sentry.io/dimagi/commcarehq/issues/702737433/

### CODE REVIEW

**Review by Rohit25negi — APPROVED** [review]

**Comment by sravfeyn** [comment]
@snopoke

File: corehq/apps/fixtures/dbaccessors.py [hunk 1]
```diff
@@ -19,14 +20,28 @@ def get_number_of_fixture_data_types_in_domain(domain):
...
+            descending=True,
+        ))
+
+    # band-aid workaround for a weird couch issue
+    #   where a deleted doc pops up in results as a dict
+    #   https://sentry.io/dimagi/commcarehq/issues/702737433/events/38122183220/
+    results = get_results()
+    for doc in results:
+        if type(doc) == dict:
+            # hopefully retry fixes it
```
Comment (snopoke) [inline corehq/apps/fixtures/dbaccessors.py]: not sure that this is the best approach, a second call could have the same issue. Maybe better to just filter out the bad docs

File: corehq/apps/fixtures/dbaccessors.py [hunk 2]
```diff
@@ -19,14 +20,28 @@ def get_number_of_fixture_data_types_in_domain(domain):
...
+            include_docs=True,
+            descending=True,
+        ))
+
+    # band-aid workaround for a weird couch issue
+    #   where a deleted doc pops up in results as a dict
+    #   https://sentry.io/dimagi/commcarehq/issues/702737433/events/38122183220/
+    results = get_results()
+    for doc in results:
+        if type(doc) == dict:
```
Comment (snopoke) [inline corehq/apps/fixtures/dbaccessors.py]: rather do `not isinstance(doc, FixtureDataType)`

File: corehq/apps/fixtures/dbaccessors.py (hunk 1 above)
Comment (sravfeyn) [reply corehq/apps/fixtures/dbaccessors.py]: Updated

**Review by snopoke — APPROVED** [review]

**Pull request closed without merging**